.github/DISCUSSION_TEMPLATE/new.yml

```
@@ -0,0 +1,61 @@
+name: New Prisma Discussion
+description: Please fill out the following information to create a new discussion related to Prisma.
+  Make sure to provide clear and concise details to help others understand and address your concerns effectively.
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description of the Idea / Problem
```

we are primarily framing GitHub Discussion as a place for "Questions", so I think we should use this as our primary wording:

```suggestion
      label: Question
```